### PR TITLE
Less warning for indent

### PR DESCRIPTION
**Pull request by gmaurel**
Part 2

### CODE REVIEW

**Comment by gmaurel** [comment]
All the statements with:
```
cpd.settings[UO_xyz].
```
need to be reviewed to clear if it is better to use
```
cpd.settings[UO_xyz].n
```
or to use
```
cpd.settings[UO_xyz].u
```
if the value of the option 'xyz' might get negative values.
The related declarations of variables are also reviewed.

File: src/indent.cpp [hunk 1]
```diff
@@ -430,7 +427,7 @@ static void indent_pse_pop(parse_frame_t &frm, chunk_t *pc)
    {
       if (pc != NULL)
       {
-         LOG_FMT(LINDPSE, "%4zu] (pp=%d) CLOSE [%zu,%s] on %s, started on line %d, level=%zu/%zu\n",
+         LOG_FMT(LINDPSE, "%4zu] (pp=%d) CLOSE [%zu,%s] on %s, started on line %zu, level=%zu/%zu\n",
```
Comment (bengardner) [inline src/indent.cpp]: The format "z" is for size_t.  The more portable method is to use the PRIuS macro as found in inttypes.h.

"...starting on line " PRIuS ", level..."

Comment (gmaurel) [inline src/indent.cpp]: With this PR I have changed 'frm.pse[frm.pse_tos].open_line' to a member of type size_t.

Using your proposal I get an compiler error, I don't understand . Can you help?
```
/home/guy-kde/Software/uncrustify/src/indent.cpp:431:39: error: expected ‘)’ before ‘PRIuS’
          printf("started on line %30" PRIuS "\n", frm.pse[frm.pse_tos].open_line);
```

Comment (jibsen) [inline src/indent.cpp]: I was not aware of any PRIuS macro part of the standard (be it C, C++ or POSIX). Perhaps it's something new?

**Pull request merged**